.sti/bin/run

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+function is_puma_installed() {
+  [ ! -f Gemfile.lock ] && return 1
+  grep ' puma ' Gemfile.lock >/dev/null
+}
+
+set -e
+
+function check_number() {
+  if [[ ! "$2" =~ ^[0-9]+$ ]]; then
+    echo "$1 needs to be a non-negative number"
+    exit 1
+  fi
+}
+check_number PUMA_WORKERS     "${PUMA_WORKERS:-0}"
+check_number PUMA_MIN_THREADS "${PUMA_MIN_THREADS:-0}"
+check_number PUMA_MAX_THREADS "${PUMA_MAX_THREADS:-0}"
+
+export RACK_ENV=${RACK_ENV:-"production"}
+
+if bundle exec rake db:migrate >/dev/null; then
+   echo "DB Migrated"
+else
+  echo "ERROR RUNNING DB MIGRATE"
+fi
+
+if is_puma_installed; then
+  export_vars=$(cgroup-limits) ; export $export_vars
+
+  exec bundle exec "puma --config ../etc/puma.cfg"
```

Comment:
This is a direct copy of the ruby sti run script that just contains an update to run the db migrations, everything else is the same.  So this is the way that the builder would expect things to be setup.